applications/GeoMechanicsApplication/custom_geometries/interface_geometry.h

```diff
@@ -284,17 +286,18 @@ class InterfaceGeometry : public Geometry<Node>
     {
         const auto points = this->Points();
 
-        // The first side coincides with the first half of the element nodes
+        // The first side is defined by the first half of the element nodes
         auto begin_of_second_side = points.ptr_begin() + (points.size() / 2);
         const auto nodes_of_first_side = PointerVector<Node>{points.ptr_begin(), begin_of_second_side};
 
-        // The second side coincides with the second half of the element nodes. However, the nodes
-        // must be traversed in opposite direction.
+        // The second side is defined by with the second half of the element nodes. However, the
```

Comment:
Sorry for the nagging:
```suggestion
        // The second side is defined by the second half of the element nodes. However, the
```

Reply:
Done